### PR TITLE
C12 hw basic styles

### PR DESCRIPTION
**Pull request by PhillipLam89**

### CODE REVIEW

**Comment by PhillipLam89** [comment]
followed specs on Figma even though it looks weird on current site

File: web/themes/custom/port_theme/scss/style.scss [hunk 1]
```diff
@@ -1 +1,35 @@
-body { background: red; }
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,700;0,900;1,100&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: tomato !important;
```
Comment (ashabed) [inline web/themes/custom/port_theme/scss/style.scss:7]: @PhillipLam89 Did you mean to leave "tomato" as the site's background color?

File: web/themes/custom/port_theme/scss/style.scss [hunk 2]
```diff
@@ -1 +1,35 @@
...
+}
+body {
+  font-family: 'Roboto', sans-serif;
+  color: #252525;
+}
+h1,h2,h3,h4,h5,h6 {
+  font-weight: 500;
+  line-height: 14px;
```
Comment (ashabed) [inline web/themes/custom/port_theme/scss/style.scss:15]: I see the misunderstanding here - it looks like you clicked "h1" in Figma and copied the style, which is reasonable. But the problem is "h1" is just a label, and by clicking on it directly you're getting the style of labels in figma. H1's should actually be styled to look like the text which appears to the right of that "h1" label. So you should actually be clicking the styled text to the right of the h1 label, and applying _that_ style.

Comment (PhillipLam89) [reply web/themes/custom/port_theme/scss/style.scss:15]: Thank you for clarifying, will fix everything once I'm home

**Comment by ashabed** [comment]
Closing to clean up older PRs - keep up the great work :)

**Pull request closed without merging**